test/services/sync/strategies/LDAPSyncerConsumer.test.js

```diff
@@ -0,0 +1,218 @@
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+const sinon = require('sinon');
+const { ObjectId } = require('mongoose').Types;
+
+const { LDAP_SYNC_ACTIONS } = require('../../../../src/services/sync/strategies/LDAPSyncer');
+const { LDAPSyncerConsumer } = require('../../../../src/services/sync/strategies/LDAPSyncerConsumer');
+const { BadRequest } = require('../../../../src/errors');
+const { SchoolRepo, ClassRepo, UserRepo } = require('../../../../src/services/sync/repo');
+
+const appPromise = require('../../../../src/app');
+
+const { expect } = chai;
+chai.use(chaiAsPromised);
+
+describe('Ldap Syncer Consumer', () => {
+	let app;
+	let sandbox;
+
+	before(async () => {
+		app = await appPromise;
+	});
+
+	beforeEach(() => {
+		sandbox = sinon.createSandbox();
```

Comment:
As far as I know, sinon has already a default sandbox, so I think we don't need a special one here.